<?xml version="1.0" encoding="UTF-8"?>
<launch>
   <include file="$(find colorblob_tracker)/launch/camera.launch"/>

   <rosparam command="load" file="$(find colorblob_tracker)/launch/param/red.yaml" />
   <node pkg="colorblob_tracker" type="color_blob_tracker" name="red_colorblob_tracker" />
</launch>
